src/modules/action-button/action-button.component.scss

```diff
@@ -0,0 +1,29 @@
+/deep/
+.stache-action-buttons {
+    display: inline-flex;
+    flex-wrap: wrap;
+
+  sky-action-button {
+    display: inline-flex;
+    flex: 0 1 20%;
+    @media (max-width: 992px) {
```

Comment:
Did you already attempt to use `StacheColumnComponent` for the breakpoints, in your HTML template? It'd be nice if we could keep the media query "logic" isolated.